src/plugins/embeddable/public/lib/panel/embeddable_panel.tsx

```diff
@@ -257,12 +258,15 @@ export class EmbeddablePanel extends React.Component<Props, State> {
   public componentDidMount() {
     if (this.embeddableRoot.current) {
       this.subscription.add(
-        this.props.embeddable.getOutput$().subscribe((output: EmbeddableOutput) => {
-          this.setState({
-            error: output.error,
-            loading: output.loading,
-          });
-        })
+        this.props.embeddable
+          .getOutput$()
+          .pipe(catchError(() => of({})))
```

Comment:
`of({})` is basically saying `continue with output = {}` . is this the intention? if we want to ignore error, I think here would make more sense to put an empty error handler to `subscribe` instead?

Reply:
good suggestion, I've implemented it!